lib/tasks/statistics.rake

```diff
@@ -0,0 +1,44 @@
+namespace :statistics do
+  desc "Get information on all form responses for a specified day, e.g. rake statistics:form_responses[\"2020-04-13\"]"
+  task :form_responses, [:date] => [:environment] do |_, args|
+    # Counts responses for each question (as defined in locales) on a given day
+    # and produces output in the following format (fictional data used as an
+    # example):
+    #
+    # What do you need to find help with because of coronavirus?
+    # - ["Feeling unsafe"]: 1
+    # - ["Feeling unsafe", "Paying bills"]: 3
+    # - ["Getting food"]: 2
+    # - ["Going in to work"]: 1
+    # - ["Having somewhere to live"]: 1
+    # - ["Paying bills"]: 3
+    # - ["Paying bills", "I’m not sure"]: 1
+    # - ["Paying bills", "Mental health and wellbeing"]: 1
+    #
+    # Do you need urgent medical help?
+    #  - No: 13
+    #
+    # Do you feel safe where you live?
+    #  - No: 2
+    #  - Yes: 1
+    #  - Yes, but I’m concerned about the safety of someone else: 1
+
+    args.with_defaults(date: Date.yesterday.to_s)
+
+    responses = FormResponse.where(created_at: Date.parse(args.date).all_day)
+
+    questions = I18n.t("coronavirus_form.groups").map { |_, v| v[:questions] }.compact.reduce(:merge)
```

Comment:
The vulnerable people form doesn't have any groups. So I think this can be: `I18n.t("coronavirus_form.questions")`

Reply:
This is for the find support form 😁

Reply:
🤦

Reply:
Ignore me! But the point about tests still stands _she says saving herself_

Reply:
If you're getting all questions from all groups, which I think you are, then you don't need `.compact.reduce(:merge)` because that gets rid of the `nils`. We used it whne excluding groups without a title.

Reply:
It looks like I don't need the `compact`, but I do need the `reduce(:merge)`, otherwise we can get an array containing a single element (a hash of all the questions).